webviz_subsurface/plugins/_map_viewer_fmu/callbacks.py

```diff
@@ -830,8 +830,8 @@ def combine_selector_values_and_name(
     ) -> dict:
         """Combine selector values with selector name for given view"""
         return {
-            id_values["selector"]: values
-            for values, id_values in zip(values, id_list)
+            id_values["selector"]: val
+            for val, id_values in zip(values, id_list)
```

Comment:
@HansKallekleiv @tnatt Am I doing the correct change here wrt. expected behaviour? Motivation was to fix the following new pylint error:
```
webviz_subsurface/plugins/_map_viewer_fmu/callbacks.py:829:8: W0613: Unused argument 'values' (unused-argument)
```

Reply:
That should work @anders-kiaer 👍